pkg/enqueue/Client/Driver/RabbitMqDriver.php

```diff
@@ -12,8 +12,21 @@ class RabbitMqDriver extends AmqpDriver
      */
     protected function doCreateQueue(string $transportQueueName): InteropQueue
     {
+        $config = parent::getConfig();
         $queue = parent::doCreateQueue($transportQueueName);
+
+        $driverOptions = $config->getDriverOptions();
+        $islazyQueue = false;
+        $lazyQueueArray = array_filter($driverOptions, function ($arr) {
+            return array_key_exists('rabbit_mq_lazy_queues', $arr);
+        });
+
+        $islazyQueue = array_key_exists($transportQueueName, $lazyQueueArray[0]['rabbit_mq_lazy_queues']);
```

Comment:
```php
$lazyQueueArray = $driverOptions['rabbit_mq_lazy_queues'] ?? [];
$isLazyQueue = $lazyQueueArray[$transportQueueName] ?? false;
```
A lot nicer and less error prone. Original code with `array_filter` can potentially break if `rabbit_mq_lazy_queues` options is not on first position and generally works slower due to the fact that `array_key_exists` is known to be roughly 5x slower that `isset`.